lib/events.js

```diff
@@ -229,8 +229,25 @@ class AdobeIOEvents {
      * @param {RetryOptions} retryOptions retry options. If set to false, retry functionality will be disabled. If not set, it will use the default times
      * @returns {Promise}
      */
-    async sendEvent(event, retryOptions=true) {
+    async sendEvent(event, retryOptions) {
         const url = `${INGRESS_HOST[this.env]}/api/events`;
+        function retryOnHttpResponse(response) {
```

Comment:
It's a matter of taste, but instead of the nested function, can we turn this into a named lambda? ( `const myFunction = () => {}`)